test-integration/helpers/utils/wait.coffee

```diff
@@ -1,40 +1,63 @@
 selenium = require 'selenium-webdriver'
+_ = require 'underscore'
 
 class Wait
   constructor: (test) -> @test = test
 
-  # TODO reduce the copy pasta between for and forMultiple
-  forMultiple: (locator, ms = 60 * 1000) =>
-    locator = @test.utils.toLocator(locator)
-    locator.shouldBeVisible ?= true
-    waitUntil = if locator.shouldBeVisible then 'elementIsVisible' else 'elementIsEnabled'
+  _for: (settings) =>
```

Comment:
pulled in this commit before this was merged in master, rebased with master and now this is no longer a part of this PR